lib/events/events.go

```diff
@@ -44,6 +44,7 @@ const (
 	FolderScanProgress
 	FolderPaused
 	FolderResumed
+	FolderWatcherErrorStateChanged
```

Comment:
Can we call this FolderWatchStateChanged, just in case there are more than one stage in the future, and we could reuse the event.

Reply:
Agreed.  
And I am not complaining, just smiling slightly: https://github.com/syncthing/syncthing/pull/4962#discussion_r190689729

Reply:
ha.

I have to find things to pick on, otherwise what's the purpose of the review?